src/Symfony/Bridge/Doctrine/Messenger/MessageRecordingEntitySubscriber.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Messenger;
+
+use Doctrine\Common\EventSubscriber;
+use Doctrine\ORM\Event\PostFlushEventArgs;
+use Doctrine\ORM\Events;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Messenger\Stamp\DispatchAfterCurrentBusStamp;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ * @author Matthias Noback <matthiasnoback@gmail.com>
+ * @author Valentin Udaltsov <udaltsov.valentin@gmail.com>
+ */
+final class MessageRecordingEntitySubscriber implements EventSubscriber
```

Comment:
Actually this subscriber does dispatching, not the recording

Reply:
@Koc, agree, renamed to `DispatchEntityMessagesDoctrineSubscriber`. Do you like this name?